tests/tests.cpp

```diff
@@ -62,44 +62,13 @@ TEST(ctor, bad_library) {
 }
 
 TEST(multiple_handles, basic_test) {
-    dylib libA("./dynamic_lib", dylib::extension);
-    dylib libB("./dynamic_lib", dylib::extension);
-}
-
-TEST(dtor, mutiple_open_close) {
-    try {
-        dylib lib;
-        lib.close();
-        lib.close();
-        lib.open(std::string("./dynamic_lib") + std::string(dylib::extension));
-        lib.open("./dynamic_lib", dylib::extension);
-        EXPECT_EQ(lib.get_function<double(double, double)>("adder")(1, 1), 2);
-        lib.close();
-        lib.close();
-        lib.close();
-        lib.get_function<double(double, double)>("adder");
-        EXPECT_EQ(true, false);
-    }
-    catch (const dylib::exception &) {
-        EXPECT_EQ(true, true);
-    }
-}
-
-TEST(get_function, bad_handler) {
-    try {
-        dylib lib("./dynamic_lib", dylib::extension);
-        lib.close();
-        lib.get_function<double(double, double)>("adder");
-        EXPECT_EQ(true, false);
-    }
-    catch (const dylib::handle_error &) {
-        EXPECT_EQ(true, true);
-    }
+    dylib libA(dylib::working_directory, "dynamic_lib");
```

Comment:
I don't like all of these "working_directory" uses, nor this constant itself... 

1. It's not common to search for libraries in the current directory, you might not even want to accommodate that especially.
2. If you must - have a named constructor idiom for it, or a proxy object etc.

Reply:
Since i removed `os_library_path` and `current_directory`, i replaced that with `"./"`